### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by MargaretTechworld**
**In this repository:**
- I have added any missing JSX elements to my project (without adding extra functionality)
- My final design matched Zeplin's design (this includes all elements + the same styling)
- I used the CSS rules detailed in Zeplin to style the website.

**My design**
image

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,39 +1,173 @@
...
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.nav {
+  letter-spacing: 1.9px;
+  color: #121212;
+  font-size: 0.813rem;
+  text-decoration: none;
```
Comment (ademibowale) [inline src/App.css:26]: - [ ] Hey kudos to you, You did a great job on making this styling look awesome but there are still issues you need to address, If you take a look at the original template you will see some differences between your design and the original design 
- [ ] According to the project requirement Your final design should match Zeplin's design (this includes all elements + the same styling Kindly make your design look exactly like the original given design to make your website look professional and user-friendly you can see the screenshot image I attached below.
- [ ] your design 
Screenshot 2023-08-26 002626

- [ ] original design
Screenshot 2023-08-26 004520

**Review by ademibowale — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @MargaretTechworld , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight 🥇

- Descriptive Pull Request title and summary. ✔️
- Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html). ✔️
- Input fields have the required attribute to activate the validation ✔️
- Max-length attribute Used on the input ✔️
- Pair-programming collaborative activity ✔️
- Good use of Github flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (MargaretTechworld) [reply src/App.css:26]: **Updated design** 

image

File: src/routes/RootLayout.js [hunk 2]
```diff
@@ -1,32 +1,19 @@
...
+      <header>
+        <div className="nav-container">
+          <h1 className="logo">Bookstore CMS</h1>
+          <NavLink className="nav books" to="/">BOOKS</NavLink>
+          <NavLink className="nav category" to="category">CATEGORIES</NavLink>
+        </div>
         <div>
-          <NavLink style={nav} to="/">Books</NavLink>
-          <NavLink style={nav} to="category">Categories</NavLink>
+          <FaUser className="user" size={14} />
```
Comment (iambenkis) [inline src/routes/RootLayout.js:14]: - [ ] As requested by the previous code reviewer, your design should match  Zeplin's design. When you check the navbar of your app, the alignement is not aligned to the design. Kindly fix it 👍 

### Your design

Capture

### Zeplin's design

image

**Review by iambenkis — CHANGES_REQUESTED** [review]
#### Hi @MargaretTechworld 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

- Input fields have the required attribute to activate the validation ✔️
- Max-length attribute Used on the input ✔️ 

## Suggested changes
_Check the comments under the review._

## Optional suggestions

- [ ] [OPTIONAL] I will advice you next time to always repect the commit message convention by writting Capitalized message 😄 , it will keep your PR professional 👍 . Kindly check[ this link](https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/) for more about the git commit message covention

Capture


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by jerryowusu — APPROVED** [review]
Hi @MargaretTechworld 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


weldone

## Highlights
Made the requested changes :+1: 
Nice design :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**